app/scripts/templates/sign_in_bounced.mustache

```diff
@@ -0,0 +1,20 @@
+<div id="main-content" class="card">
+  <header>
+    <h1 id="fxa-signin-bounced-header">{{#t}}Sorry. We've locked your account.{{/t}}</h1>
+  </header>
+
+  <section>
+    <div class="graphic graphic-mail-bounced"></div>
+
+    <p>{{#t}}The confirmation email we sent to %(email)s was returned and we've locked your account to protect your Firefox data.{{/t}}</p>
+
+    <div class="extra-links">{{#unsafeTranslate}}If this is a valid email address, <a href="https://support.mozilla.org/" target="_blank">let us know</a> and we can help unlock your account.{{/unsafeTranslate}}</div>
+
+    <div class="links margin-top">
+      <a id="create-account" class="{{#canGoBack}}left {{/canGoBack}}delayed-fadein" href="#" data-flow-event="link.create-account">{{#t}}No longer own that email? Create a new account{{/t}}</a>
+      {{#canGoBack}}
+        <a id="back" class="right delayed-fadein" href="#" data-flow-event="link.back">{{#t}}Back{{/t}}</a>
```

Comment:
Just realised the `data-flow-event` attribute requires `views/mixins/flow-events-mixin`. Will add, sorry.